lib/rules/value-no-vendor-prefix/README.md

```diff
@@ -72,5 +72,20 @@ cursor: -webkit-grab;
 
 <!-- prettier-ignore -->
 ```css
-.foo { max-width: -moz-max-content; }
+.foo { list-style-type: -moz-hangul; }
 ```
+
+<!-- prettier-ignore -->
+```css
+.foo { list-style-type: -moz-hangul-consonant; }
+```
+
+<!-- prettier-ignore -->
+```css
+.foo { -webkit-appearance: -apple-pay-button; }
+```
+
+> [!WARNING]
+> An _exact_ match comparison will be performed for non-regex strings in the next major version.
+> If you want to keep the legacy behavior, please consider using a regex instead.
+> e.g. `[/^(-webkit-|-moz-|-ms-)?inline-/]`
```

Comment:
```suggestion
> An _exact_ match comparison will be performed for non-regex strings in the next major version.  
> If you want to keep the legacy behavior, please consider using a regex instead.  
> e.g. `[/^(-webkit-|-moz-|-ms-)?inline-/]`
```

Reply:
We don't care about small screens and already have long lines, for example, see L14 in this file.

In addition, our Editorconfig settings may remove trailing whitespaces. So, it seems better to avoid trailing whitespaces as possible:

https://github.com/stylelint/stylelint/blob/8c23fcecc93edb3be32eaad313a7270fae6451e4/.editorconfig#L9

Reply:
Okay, how about this? https://github.com/stylelint/stylelint/pull/7650/files#r1579504581